src/main/webapp/script.js

```diff
@@ -47,22 +51,80 @@ function initializeRow(json, key, currentRow) {
   let index = 0;
   for (const type of json[key]) {
     const columnEntry = document.createElement('td');
-    console.log(type);
-    console.log(type.toUpperCase());
+
     if ((type.toUpperCase() === 'POSE' && index === 0) ||
       (type.toUpperCase() === 'POINTCLOUD' && index === 1)) {
       const link = document.createElement('a');
       link.href = '/getrun?id=' + key + '&dataType=' + json[key];
       link.innerText = type;
       columnEntry.appendChild(link);
+      currentRow.appendChild(columnEntry);
+      /* Makes checkboxes for each data entry. */
+      const checkbox = document.createElement('INPUT');
+      checkbox.type = 'checkbox';
+      checkbox.setAttribute('name', 'check');
+      checkbox.setAttribute('value', link.getAttribute('name'));
+      currentRow.appendChild(checkbox);
     } else {
       columnEntry.innerText = '';
+      currentRow.appendChild(columnEntry);
     }
-    currentRow.appendChild(columnEntry);
     index++;
   }
   return currentRow;
 }
+
+/* Select all Checkboxes function. */
+function toggle(source) {
```

Comment:
toggle and json called in index page which is already merged to master with changes. todo: delete makemultirunurl since it is replaced by json function